src/Interfaces/LazySet.jl

```diff
@@ -1500,3 +1501,19 @@ rationalize(::Type{T}, X::LazySet{N}; tol::Real=eps(N)) where {T<:Integer, N<:Ab
 # method extension for lazy sets
 _rationalize(::Type{T}, X::AbstractVector{<:LazySet{N}}, tol::Real) where {T<:Integer, N<:AbstractFloat} = rationalize.(Ref(T), X, Ref(tol))
 _rationalize(::Type{T}, X::LazySet{N}, tol::Real) where {T<:Integer, N<:AbstractFloat} = rationalize(T, X, tol)
+
+"""
+    permute(X::LazySet, p::AbstractVector{Int})
+
+Permute the dimensions of a set according to a given permutation vector.
+
+### Input
+
+- `X` -- set
+- `p` -- permutation vector
+
+### Output
+
+A new equivalent set where the dimensions have been permuted.
```

Comment:
I think that the resulting set is not "equivalent" in the sense of `isequivalent`, so should we use another word here?